addons/google_calendar/tests/test_sync_odoo2google.py

```diff
@@ -107,7 +107,7 @@ def test_recurring_event_creation_perf(self):
             'duration': 18,
         })
         partner_model = self.env.ref('base.model_res_partner')
-        with self.assertQueryCount(__system__=86):
+        with self.assertQueryCount(__system__=794):
```

Comment:
Isn't it insane that we get 9x the queries from this? Is this correct? Do we have multiple loops without prefetch?

Reply:
Basically it was not working before. The code was completely false, and skipped activities entirely. And you can't prefetch due to calendar event creating activities in a o2m instead of preparing them in batch (will be improved by Arnaud's team).

Reply:
Hm makes sense I guess if it skipped most of them